docs/Verilog.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title:  Introduction to Verilog Programming
+nav_order: 
```

Comment:
Please specify a nav order, otherwise it defaults to the home page. You wouldn't want that, right? Judge a particular page number where this would fit. Then change all the numbers following that. That's a trouble I know, but needed.

Reply:
outdated.